mobile-save-for-later/conf/cfn.yaml

```diff
@@ -1,279 +1,307 @@
+    Type: 'AWS::ApiGateway::DomainName'
+    Properties:
+      CertificateArn: !Ref CertArn
+      DomainName: !Sub '${App}.${HostedZoneName}'
   FetchArticlesLambda:
-    Type: AWS::Serverless::Function
+    Type: 'AWS::Lambda::Function'
     Properties:
-      Handler: com.gu.sfl.lambda.FetchArticlesLambda::handleRequest
-      Role: !GetAtt SaveForLaterRole.Arn
-      Runtime: java8
-      CodeUri:
-        Bucket: !Ref DeployBucket
-        Key: !Sub ${Stack}/${Stage}/${App}/${App}.jar
-      FunctionName: !Sub ${App}-FETCH-${Stage}
+      Code:
+        S3Bucket: !Ref DeployBucket
+        S3Key: !Sub '${Stack}/${Stage}/${App}/${App}.jar'
+      FunctionName: !Sub '${App}-FETCH-${Stage}'
+      Tags:
+        - Value: SAM
+          Key: 'lambda:createdBy'
+      ReservedConcurrentExecutions: !FindInMap
+        - StageVariables
+        - !Ref Stage
+        - ReservedConcurrency
+      MemorySize: 394
       Environment:
         Variables:
-          Stack: !Ref Stack
-          Stage: !Ref Stage
           App: !Ref App
+          Stage: !Ref Stage
+          Stack: !Ref Stack
           IdentityApiHost: !Ref IdentityApiHost
-      MemorySize: 394
+      Handler: 'com.gu.sfl.lambda.FetchArticlesLambda::handleRequest'
+      Role: !GetAtt SaveForLaterRole.Arn
       Timeout: 20
-      ReservedConcurrentExecutions: !FindInMap [ StageVariables, !Ref Stage, ReservedConcurrency ]
-      Events:
-        GetApi:
-          Type: Api
-          Properties:
-           Path: "/syncedPrefs/me"
-           Method: GET
-           RestApiId: !Ref SaveForLaterApi
-
-  SaveForLaterReadThrottleEvents:
-    Type: AWS::CloudWatch::Alarm
-    Properties:
-      AlarmName: !Sub ${App}-${Stage}-articles-ReadCapacityUnitsLimit-BasicAlarm
-      Namespace: AWS/DynamoDB
-      MetricName: ReadThrottleEvents
-      Unit: Count
-      Statistic: Sum
-      ComparisonOperator: GreaterThanThreshold
-      Dimensions:
-        - Name: TableName
-          Value: !Ref SaveForLaterDynamoTable
-      Threshold: 0
-      Period: 60
-      EvaluationPeriods: 1
-      TreatMissingData: notBreaching
-      ActionsEnabled: !FindInMap [StageVariables, !Ref Stage, AlarmActionsEnabled]
-      AlarmActions: [ !Ref DynamoNotificationTopic ]
-
-  SaveForLaterWriteThrottleEvents:
-    Type: AWS::CloudWatch::Alarm
-    Properties:
-      AlarmName: !Sub ${App}-${Stage}-articles-WriteCapacityUnitsLimit-BasicAlarm
-      Namespace: AWS/DynamoDB
-      MetricName: WriteThrottleEvents
-      Unit: Count
-      Statistic: Sum
-      ComparisonOperator: GreaterThanThreshold
-      Dimensions:
-        - Name: TableName
-          Value: !Ref SaveForLaterDynamoTable
-      Threshold: 0
-      Period: 60
-      EvaluationPeriods: 1
-      TreatMissingData: notBreaching
-      ActionsEnabled: !FindInMap [StageVariables, !Ref Stage, AlarmActionsEnabled]
-      AlarmActions: [ !Ref DynamoNotificationTopic ]
+      Runtime: java8
+Description: Implements save for later for mobile
```

Comment:
minor: The diff here looks bigger than it is as the template has been reordered, placing `Resources` at the top. Is it worth moving them to the bottom again? (IIUC this re-ordering is the result of downloading the transformed template from AWS).

Reply:
Although the diff is ugly I would prefer not to manipulate this file by hand since it was machine-generated (I downloaded it from AWS after the `SAM` transformation) - I think the change set is more useful than a diff here anyway. I will talk the Mobile team through that (we're going to review this work in person)